MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -121,6 +121,10 @@ open class InstructionsCardViewController: UIViewController {
     
     @objc func orientationDidChange(_ notification: Notification) {
```

Comment:
Overall looks good. Thanks. One thing I've noticed is that on iPad `UIDevice.orientationDidChangeNotification` does not return correct UIScreen size, this leads to incorrect size of guidance card itself. You can try to replace it with such callback to see if it helps:
```swift
override public func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
    super.viewWillTransition(to: size, with: coordinator)
}
```

This can also be done in either this PR or another one.
<img width="350" alt="Screen Shot 2020-12-01 at 7 10 40 PM" src="https://user-images.githubusercontent.com/1496498/100823448-1533d300-3409-11eb-8c99-e65972f7c2d5.png">

Reply:
I will handle this in a separate ticket. I created a branch for it already.